components/schemes/Typography/src/MDCTypographyScheme.h

```diff
@@ -0,0 +1,72 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+
+/**
+ A simple typography scheme that provides semantic fonts. There are no optional
+ properties, all fonts must be provided, supporting more reliable typography theming.
+ */
+@protocol MDCTypographyScheming
+
+@property(nonatomic, nonnull, readonly) UIFont *headline1;
```

Comment:
Please provide documentation for these properties.

Reply:
@ajsecord is there documentation snippets for these APIs?

Reply:
Er, this is getting close to an emergency in terms of our release, so let's prioritize getting this in and add docs later. Add an issue and assign to the original author?

Reply:
+1

https://www.pivotaltracker.com/story/show/156555861